pkg/console/util.go

```diff
@@ -101,7 +101,7 @@ func validateNTPServers(ntpServerList []string) error {
 		if err != nil {
 			return err
 		}
-		if err := conn.SetDeadline(time.Now().Add(5 * time.Second)); err != nil {
+		if err := conn.SetDeadline(time.Now().Add(15 * time.Second)); err != nil {
```

Comment:
I reckon 3 retries with 5 seconds timeout should be sufficient enough, otherwise, it takes too long for the NTP config from the user perspective.

Reply:
Updated it.